x-pack/plugins/maps/common/descriptor_types/layer_descriptor_types.ts

```diff
@@ -46,6 +46,7 @@ export type LayerDescriptor = {
   visible?: boolean;
   style?: StyleDescriptor | null;
   query?: Query;
+  isFittable?: boolean;
```

Comment:
isFittable is really a function of the layer source, layer visibility, and whether this flag is set. How about renaming to `includeInFitToBounds`?